install.sh

```diff
@@ -291,7 +291,10 @@ nvm_do_install() {
   local NVM_PROFILE
   NVM_PROFILE="$(nvm_detect_profile)"
   local INSTALL_DIR
-  INSTALL_DIR="$(nvm_install_dir)"
+  INSTALL_DIR="$(
+    nvm_install_dir | 
+    command sed s:^$HOME:\$HOME:
```

Comment:
I didn't want to affect other functions which use `nvm_install_dir` - I was intentionally trying to isolate this change to the place that needed it. I'm testing it now.